src/routes/transactions/transactions-view.service.ts

```diff
@@ -287,12 +293,25 @@ export class TransactionsViewService {
       depositExecutionDate: null,
     });
     const value = args.value ? Number(args.value) : 0;
+    const chain = await this.chainsRepository.getChain(args.chainId);
+    const nativeCoinPrice =
```

Comment:
The `NetworkStats` from Kiln include a `eth_price_usd` value that we could use instead of fetching this. If we cache the response, it should save the extra request. What do you think?

Reply:
Oh, I didn't see that! Excellent point, will change it! Thanks!